go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ssvlabs/ssv
 
-go 1.22
+go 1.23.1
```

Comment:
This is pretty much a requirement to use `tailscale.com v1.76.3`

Reply:
I've added [a commit on top](https://github.com/ssvlabs/ssv/pull/1840/commits/5d14e31ff5a3f8d7621a09b3fae17af5106a5e30) to drop it down to `tailscale.com v1.72.0` so we can keep using `go 1.22` like this (as I've done in https://github.com/ssvlabs/ssv/pull/1798#discussion_r1809543763):

```
go 1.22.0

toolchain go1.23.1
```